docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -143,6 +143,75 @@ game.properties:        158 bytes
 ui.properties:          83 bytes
 ```
 
+You can pass in the `--from-env-file` to create a ConfigMap from an env-file (i.e. a Docker .env file), for example:
```

Comment:
this would be simpler and more easily understood as 

> Use the option `--from-env-file` to create

Also please replace (i.e.) with written english - we try not to use latin phrases per our style guide: https://kubernetes.io/docs/home/contribute/style-guide/#avoid-latin-phrases.